turbo/cli/flags.go

```diff
@@ -157,6 +157,12 @@ var (
 		Name:  "healthcheck",
 		Usage: "Enable grpc health check",
 	}
+
+	JWTSecretPath = cli.StringFlag{
```

Comment:
Why is it defined in 2 places?

Reply:
I deleted this one. I will use the one inside of utils for consistency.